### PR TITLE
COMMERCE-9691 Fix minicart item quantity popover clipping

### PR DESCRIPTION
**Pull request by dsitu**
https://issues.liferay.com/browse/COMMERCE-9691

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/_mini_cart_item.scss [hunk 1]
```diff
@@ -10,7 +10,7 @@
 		'info quantity price delete'
 		'errors errors errors errors';
 	grid-template-columns: auto 90px minmax(min-content, 100px) min-content;
-	overflow: hidden;
+	overflow: visible;
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/_mini_cart_item.scss:13]: ehi @dsitu what is the effect of this? does the scrollbar get visible? If yes I believe that maybe the right solution would be to work on the z-index of the popover.

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/_mini_cart_item.scss (hunk 1 above)
Comment (dsitu) [reply modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/_mini_cart_item.scss:13]: Hey @andrea-ale-sbarra, I tested this again and the scroll bar is **not** visible when the product name is longer. The product name gets cut off with "...". The popover shows up now and is shown over the cart items.

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
QA Failed

**Pull request closed without merging**